README.md

```diff
@@ -4,37 +4,68 @@
 [![Discord](https://img.shields.io/badge/chat-on%20discord-brightgreen.svg)](https://discord.gg/GsEFRM8)
 [![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/prismarine-web-client)
 
-A minecraft client running in a web page. Demo at https://prismarine-web-client.js.org
+A Minecraft client running in a web page.
 
-It runs mineflayer in the browser which connects to a websocket minecraft server.
-It provides a simple websocket to tcp proxy as a backend to make it possible to connect to any minecraft server.
+## How it Works
+prismarine-web-client runs mineflayer and prismarine-viewer in the browser, which connects over WebSocket to a proxy which translates the WebSocket connection into TCP to connect to normal Minecraft servers.
 
 ## Screenshot
-![Screenshot of MineWeb in action](screenshot.png)
+![Screenshot of prismarine-web-client in action](screenshot.png)
 
-## Usage
+## Live Demo
+Click on this link to open it in your browser, no installation necessary: https://prismarine-web-client.js.org
 
-`npm install -g prismarine-web-client` then run `prismarine-web-client` then open `http://localhost:8080` in your browser
+*Note: Don't open it on iOS. It doesn't work. You can try it on Android, which sometimes works and sometimes doesn't. It's best open it on a laptop/desktop though.*
```

Comment:
Isn't very useful in the readme. Let's just fix it

Reply:
Won't people get confused on why it isn't working though, possibly making them think that it's broken? I think it might be worth leaving it here so people know not to try on mobile.

Reply:
> About mobile just put explicitly which platform works

Done.